RebootServer.py

```diff
@@ -176,8 +176,10 @@ def RunServer(miners):
                         print("miner %s's health check passed!" % miners[miner].name, flush=True)
                         pass
                     # End if/else block
-                #End if/else block
-            # End for
+                #End for block
+            else:
+                print("Connection to Google not found. Internet may be down.", flush=True)
+			# End if/else block
```

Comment:
Why so much indentation here? Shouldn't this comment match up with the `else` right above it?